mizar/dp/mizar/operators/endpoints/endpoints_operator.py

```diff
@@ -258,12 +274,14 @@ def create_simple_endpoints(self, interfaces, spec):
 
             ep.set_vni(spec['vni'])
             ep.set_vpc(spec['vpc'])
-            ep.set_net(spec['net'])
+            ep.set_net(net_info.get('subnet', spec['net']))
```

Comment:
With changing the kind of the net crd to "subnet" we will need to look into spec["subnet"] here.